modules/plugins/jboss-as-7/src/main/java/org/rhq/modules/plugins/jbossas7/helper/TruststoreConfig.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * RHQ Management Platform
+ * Copyright (C) 2005-2015 Red Hat, Inc.
+ * All rights reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation version 2 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA
+ */
+package org.rhq.modules.plugins.jbossas7.helper;
+
+import org.w3c.dom.Node;
+
+/**
+ * 
+ * @author lzoubek
+ *
+ */
+public class TruststoreConfig {
```

Comment:
Not very object oriented. Could you make the fields private final, use a private constructor and add getters?